test/gtest/ucp/test_ucp_sockaddr.cc

```diff
@@ -357,8 +366,7 @@ class test_ucp_sockaddr : public ucp_test {
     void listen_and_communicate(ucp_test_base::entity::listen_cb_type_t cb_type,
                                 bool wakeup)
     {
-        struct sockaddr_in connect_addr;
-        get_listen_addr(&connect_addr);
+        struct sockaddr_in connect_addr = test_addr;
```

Comment:
how is it related to adding skip condition?

Reply:
`get_sockaddr()` is called by `test_ucp_sockaddr::init()` function
If `get_sockaddr()` can't find an address, it skip the test, otherwise sets `test_addr` filed to the address found

Reply:
ok. so can we remove the local variable connect_addr ?

Reply:
yes, we can - done. we can't do this only for 451 line because `listen_addr` is changed